### ISSUE TITLE
dot.gnuplot has a hard coded library path non universal

### ISSUE THREAD

**Issue by csrabak** [janert/gnuplot-in-action-2e#1]
In line 11 of `dot.gnuplot` file the loadpath points to `"/home/janert/lib/gnuplot/"`, which is specific and non canonical in any of the gnuplot supported platforms.

**Comment by janert** [comment]
That's right. It's a straw-man example; I expect other people to adapt it to their situation.

As I explain on p276 of my book, this is (actually: was) my own dot-gnuplot file, and is shown as an example for the typical contents of an individual's config file. Naturally, other users will have their own preferences and local configs, including paths. 

To my knowledge, there are no "canonical" paths for personalized gnuplot snippets (and none for data files!); everyone will have to make their own decisions and set up their local configs accordingly.

**Comment by csrabak** [comment]
I see and hear you, may I then suggest the example file have in addition to the book passage, a comment line explicitly explaining this.

Also, an information if the absence of any 'individual's config file' would allow the example file to run or not?
